src/regions/ca/info/ca-bc/reporting.fr.md

```diff
@@ -1,14 +1,22 @@
 ## Colombie-Britannique: Que dois-je faire si je pense que j'ai COVID-19 ?
 
-Si vous développez des symptômes de rhume, de grippe ou de [type COVID-19 (en anglais seulement)](http://www.bccdc.ca/health-info/diseases-conditions/covid-19/about-covid-19/symptoms), utilisez l'[outil d'auto-évaluation COVID-19 de la Colombie-Britannique (en anglais seulement)](https://bc.thrive.health/) pour vous aider à déterminer si vous avez besoin d'une évaluation plus approfondie pour le test COVID-19 par un médecin, une infirmière praticienne ou dans un centre de collecte local. Vous pouvez effectuer cette évaluation pour vous-même, ou au nom d'une autre personne, si elle n'est pas en mesure de le faire.
+_Les liens sont disponibles en anglais seulement._
 
-Consultez la page [Si vous êtes malade (en anglais seulement)](http://www.bccdc.ca/health-info/diseases-conditions/covid-19/about-covid-19/if-you-are-sick) pour savoir comment arrêter la propagation des germes, ce qu'il faut faire si vos symptômes s'aggravent et comment mettre fin à l'auto-isolement.
+Si vous développez des symptômes de rhume, de grippe ou de [type COVID-19](http://www.bccdc.ca/health-info/diseases-conditions/covid-19/about-covid-19/symptoms), visitez la page [Testing information](http://www.bccdc.ca/health-info/diseases-conditions/covid-19/testing) ou utilisez l'[outil d'auto-évaluation COVID-19 de la Colombie-Britannique](https://bc.thrive.health/) pour vous aider à déterminer si vous avez besoin d'une évaluation plus approfondie pour le test COVID-19 par un médecin, une infirmière praticienne ou dans un centre de collecte local. Vous pouvez effectuer cette évaluation pour vous-même, ou au nom d'une autre personne, si elle n'est pas en mesure de le faire.
 
-### Je suis un voyageur de retour, dois-je m'isoler?
+Consultez la page [Si vous êtes malade](http://www.bccdc.ca/health-info/diseases-conditions/covid-19/about-covid-19/if-you-are-sick) pour savoir comment arrêter la propagation des germes, ce qu'il faut faire si vos symptômes s'aggravent et comment mettre fin à l'auto-isolement.
 
-Oui, vous devriez vous isoler. Veuillez consulter notre page sur [l'auto-isolement (en anglais seulement)](http://www.bccdc.ca/health-info/diseases-conditions/covid-19/self-isolation) pour savoir combien de temps il faut s'isoler si vous êtes un voyageur de retour.
+### Je suis de retour en Colombie Britannique, que dois-je faire avant d'arriver?
```

Comment:
Why did we rename this heading? :)

Reply:
On utilise le titre de la section dans la page du site gouvernementale et ils ont supprimé le titre précédent pour ajouter celui-ci et mettre les informations relatif à cette question. Ça remplace un truc du genre "je suis de retour, dois-je m'auto-isoler?". le nouveau titre qu'ils ont mis sur le site correspond mieux à leur nouveau texte.